ccm_web/server/appHandler.ts

```diff
@@ -72,11 +59,8 @@ class AppHandler {
     // Set lti launch callback
     // When receiving successful LTI launch redirects to app.
     provider.onConnect(async (token: IdToken, req: Request, res: Response) => {
-      console.log(token)
-      if (!this.envOptions.isDev) {
+      console.log(token["userInfo"])
         return res.sendFile(path.join(this.envOptions.staticPath, 'index.html'))
-      }
-      return provider.redirect(res, `http://localhost:${server.clientPort}`)
     })
```

Comment:
The linter wants you to change `token["userInfo"]` to `token.userInfo` and fix the indentation in the arrow function. Looks like you were using IntelliJ, so you may need to fiddle with the ESLint configuration there.

Reply:
Done, strange my linter never complain. I have Type-script Standard JS installed in VScode. I will look into it